src/jukebox/jukebox/version.py

```diff
@@ -1,14 +1,16 @@
 
 VERSION_MAJOR = 3
-VERSION_MINOR = 4
+VERSION_MINOR = 5
 VERSION_PATCH = 0
-VERSION_EXTRA = ""
+VERSION_EXTRA = "alpha"
```

Comment:
Still alpha? Should we change it to beta?

Reply:
We just use "alpha" to differentiate the develop branch. Theres no "beta", after the merge its prod on main.